src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -53,8 +53,8 @@ public AddCommand parse(String args) throws ParseException {
         Review review = ParserUtil.parseReview(argMultimap.getValue(PREFIX_REVIEW).isPresent()
                 ? argMultimap.getValue(PREFIX_REVIEW).get() : "");
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
-        Rating rating = ParserUtil.parseRating(argMultimap.getValue(PREFIX_RATING).isPresent()
-            ? argMultimap.getValue(PREFIX_RATING).get() : "0");
+        Rating rating = argMultimap.getValue(PREFIX_RATING).isPresent()
+            ? ParserUtil.parseRating(argMultimap.getValue(PREFIX_RATING).get()) : ParserUtil.parseRating();
 
```

Comment:
Great idea to abstract the Rating Parser into ParserUtil to use throughout the app, like for edit